source/support/connect-to-govwifi-using-a-mac-imac-or-macbook.html.erb

```diff
@@ -27,17 +29,17 @@ description: Follow these instructions to connect your Mac, iMac or Macbook to G
             <%= partial 'shared/copy/password_instructions' %>
           </li>
           <li>
-            <p>Select <strong>Join</strong>. You will be asked to confirm the network is genuine by checking the certificate.</p>
+            <p>Select <strong>Join</strong>. You will be asked to confirm the network is genuine by checking the
```

Comment:
I find the line break disrupting the flow of the markup, is this just personal preference?